AdyenCard/Components/GiftCardComponent/GiftCardComponent.swift

```diff
@@ -119,7 +158,14 @@ public final class GiftCardComponent: PresentableComponent,
 
     internal lazy var securityCodeItem: FormTextInputItem = {
         let item = FormTextInputItem(style: style.textField)
-        item.title = localizedString(.cardPinTitle, localizationParameters)
+        let title: String
+        switch partialPaymentMethodType {
+        case .giftCard:
+            title = localizedString(.cardPinTitle, localizationParameters)
+        case .mealVoucher:
+            title = localizedString(.cardCvcItemTitle, localizationParameters)
+        }
```

Comment:
As a thought:

Maybe we can move this logic to **ViewModel** ?

Reply:
Resolved as discussed offline that it may not have been needed for a small difference